### DISCUSSION TITLE
404 when trying to get started with Vercel Postgres

### DISCUSSION THREAD

**Discussion by amarvinj** [vercel/community#5686] [Bug Report] [answered]
### Question

<img width="1710" alt="Screenshot 2024-01-31 at 12 58 13 PM" src="https://github.com/vercel/community/assets/122550891/fad2c9a9-05dc-44c6-82f0-2f03c053560e">
<img width="1710" alt="Screenshot 2024-01-31 at 12 59 07 PM" src="https://github.com/vercel/community/assets/122550891/8f78b5bd-4eba-45c2-99e7-7019fe266041">

i am geeting a 404 error when trying to get started with Vercel Postgres,

**Comment by amyegan** [comment]
Thanks for reporting this. I'll let the team know so we can get it fixed.

Until we have the link fixed, you can still get to your Storage page another way. Go to [vercel.com](https://vercel.com) and sign in to view your dashboard, click on the `Storage` tab, and then click the `Create Database` button to get started 🙂

**Reply by amyegan** [reply, marked as answer]
The link is fixed now 😄

[a comment by an automated account is not shown]
